src/peergos/shared/user/fs/FileWrapper.java

```diff
@@ -610,8 +655,7 @@ public static int getNumberOfChunks(long size) {
                 .collect(Collectors.toList());
     }
 
-    @JsMethod
-    public CompletableFuture<FileWrapper> uploadFileJS(String filename,
+    public CompletableFuture<FileWrapper> uploadFile(String filename,
```

Comment:
remember this change has a corresponding web-ui PR

Reply:
I've restored the method name and annotation. No change required for UI.